### PR TITLE
1.1 is done

### PR DESCRIPTION
**Pull request by dmytro-hryhoruk**

### CODE REVIEW

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 1]
```diff
@@ -15,6 +15,11 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        int returnStatement = ((result1 == bet1) && (result2 == bet2))? 2 :
+                ((result1 == result2 && bet1 == bet2
+                || result1 < result2 && bet1 < bet2
+                || result1 > result2 && bet1 > bet2)) ?
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/FootballMatch.java]: '?' should be on a new line

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 2]
```diff
@@ -15,6 +15,11 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        int returnStatement = ((result1 == bet1) && (result2 == bet2))? 2 :
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/FootballMatch.java]:
```suggestion
        int returnStatement = ((result1 == bet1) && (result2 == bet2)) ? 2 :
```

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 3]
```diff
@@ -15,6 +15,11 @@
...
+                ((result1 == result2 && bet1 == bet2
+                || result1 < result2 && bet1 < bet2
+                || result1 > result2 && bet1 > bet2)) ?
+                1: 0;
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/FootballMatch.java]:
```suggestion
                1 : 0;
```

File: src/main/java/core/basesyntax/FootballMatch.java (hunk 2 above)
Comment (Sofasmile) [inline src/main/java/core/basesyntax/FootballMatch.java]: Can we implemented this method without redundant variable?

**Review by Sofasmile — CHANGES_REQUESTED** [review]
remember about codestyle
https://mate-academy.github.io/style-guides/java/java.html

File: src/main/java/core/basesyntax/FootballMatch.java (hunk 2 above)
Comment (dmytro-hryhoruk) [reply src/main/java/core/basesyntax/FootballMatch.java]: Yes,We can.

**Review by Sofasmile — APPROVED** [review]

**Pull request closed without merging**